randovania/games/factorio/pickup_database/pickup-database.json

```diff
@@ -474,7 +474,8 @@
                 "heavy-armor"
             ],
             "preferred_location_category": "major",
-            "expected_case_for_describer": "shuffled"
+            "expected_case_for_describer": "shuffled",
+            "description": "Provides in order: Light armor \u2192 Heavy armor"
```

Comment:
IG this doesnt make sense to use as typical progressive chain?

Reply:
I don't want settings to make it non-progressive.